requirements.txt

```diff
@@ -8,3 +8,4 @@ llama_index
 torch
 transformers
 uvicorn
+black
```

Comment:
do we actually need to add this as a project requirement?  i'd prefer not to since we don't actually need it at runtime, so we don't need/want to be adding it to our published images.

Seems like something developers can just install globally on their machine

Reply:
Yes, sounds good to me.